packages/mds-types/index.ts

```diff
@@ -80,37 +122,48 @@ export const AUDIT_EVENT_TYPES = Enum('start', 'note', 'summary', 'issue', 'tele
 export type AUDIT_EVENT_TYPE = keyof typeof AUDIT_EVENT_TYPES
 
 export const EVENT_STATUS_MAP: { [P in VEHICLE_EVENT]: VEHICLE_STATUS } = {
-  register: VEHICLE_STATUSES.removed,
-  service_start: VEHICLE_STATUSES.available,
-  service_end: VEHICLE_STATUSES.unavailable,
-  provider_drop_off: VEHICLE_STATUSES.available,
-  provider_pick_up: VEHICLE_STATUSES.removed,
-  agency_pick_up: VEHICLE_STATUSES.removed,
-  agency_drop_off: VEHICLE_STATUSES.available,
-  reserve: VEHICLE_STATUSES.reserved,
-  cancel_reservation: VEHICLE_STATUSES.available,
-  trip_start: VEHICLE_STATUSES.trip,
-  trip_enter: VEHICLE_STATUSES.trip,
-  trip_leave: VEHICLE_STATUSES.elsewhere,
-  trip_end: VEHICLE_STATUSES.available,
-  deregister: VEHICLE_STATUSES.inactive
+  register: 'removed',
+  service_start: 'available',
+  service_end: 'unavailable',
+  provider_drop_off: 'available',
+  provider_pick_up: 'removed',
+  agency_pick_up: 'removed',
+  agency_drop_off: 'available',
+  reserve: 'reserved',
+  reserve_stop: 'stopped',
+  cancel_reservation: 'available',
+  trip_start: 'trip',
+  trip_enter: 'trip',
+  trip_leave: 'elsewhere',
+  trip_end: 'available',
+  deregister: 'inactive',
+  service_enter: 'available',
+  service_leave: 'elsewhere',
+  depot_enter: 'removed',
+  depot_leave: 'unavailable',
+  reserve_enter: 'reserved',
+  trip_stop: 'stopped',
+  trip_resume: 'trip'
```

Comment:
Generally speaking, I like to try to keep this sort of thing sorted. Makes it easier to find things, IMO.

Reply:
Agreed, I was hoping my formatter would auto-sort, but no dice. Might see if there's a liter rule I can enable or something for a second pass to clean it up.